Watchman.Discord/WatchmanBot.cs

```diff
@@ -68,6 +67,19 @@ private Task MessageReceived(SocketMessage message)
         //    return this._workflow.Run(msg);
         //}
 
+        private void LogException(Exception e, SocketMessage socketMessage)
+        {
+            var messagesService = new MessagesService()
+            {
+                DefaultChannelId = socketMessage.Channel.Id
+            };
+
+            if (e is Exception)
+            {
+                messagesService.SendMessage("Bot nie ma wystarczających uprawnień do zarządzania rolami użytkowników");
```

Comment:
to bym dał bardziej ogólne => później i tak będzie to brane z zewnątrz więc na razie nie przeszkadza 

ale zmień na bardziej ogólny tekst żeby nie myliło przy debugowaniu